src/accumulator/stump.rs

```diff
@@ -0,0 +1,139 @@
+use super::types;
```

Comment:
There's trailing spaces in these lines that should be removed: 18, 41, 45, 47, 48, 69, 94, 120, 135.